docs/models/blockstates/forgeBlockstates.md

```diff
@@ -0,0 +1,140 @@
+Forge's Blockstates
+===================
+
+Forge has its own blockstate json format to accommodate for modders needs. It introduces submodels, which allows you to build the final blockstate from different parts.
+
+!!! Attention
+
+    Note that all models and textures referenced are from vanilla minecraft. For your own mod, you have to use the full location! For example: `"mymod:blocks/blockTexture"`.
+
+    You don't have to use Forge's blockstate format at all, you can also use the vanilla format!
+
+General Structure of the Format
+-------------------------------
+
+```json
+{
+  "forge_marker": 1,
+  "defaults": {
+    "textures": {
+      "all": "blocks/dirt"
+    },
+    "model": "cube_all",
+    "uvlock": true
+  },
+  "variants": {
+    "normal": [{}]
+  }
+}
+```
+
```

Comment:
Could you please include the information from Jay Avery's very informative post [here](http://www.minecraftforge.net/forum/topic/57153-1102-issue-with-fluid-json-files/?do=findComment&comment=266664), regarding fully defined and partially defined variants.